ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/transaction/filtered/UtxoFilteredTransactionBuilder.java

```diff
@@ -178,6 +179,19 @@ public interface UtxoFilteredTransactionBuilder {
     @Suspendable
     UtxoFilteredTransactionBuilder withOutputStates(@NotNull Predicate<ContractState> predicate);
 
+    /**
+     * Includes an audit proof of the output state refs component group from a {@link UtxoSignedTransaction} in the
+     * current {@link UtxoFilteredTransaction}.
+     *
+     * @param indexes index of transaction to include its output to {@link UtxoFilteredTransaction}
```

Comment:
Is this correct like this?

Reply:
could you elaborate on your question?

Reply:
Is it not `indexes of the output states to be included` or something like that?
(But I might have misunderstood something.)

Reply:
Ah, yeah it should be plural